.editorconfig

```diff
@@ -14,5 +14,4 @@ trim_trailing_whitespace = true
 end_of_line = crlf
 
 [*.yml]
-indent_style = space
```

Comment:
This needs to be retained, YAML files need to be space indented.

Reply:
No, it is inherited. See IDE that Highlights this with the right plugin.